components/esp_lvgl_port/test_apps/simd/main/test_lv_fill_functionality.c

```diff
@@ -81,6 +82,13 @@ static void test_eval_32bit_data(func_test_case_params_t *test_case);
  */
 static void test_eval_16bit_data(func_test_case_params_t *test_case);
 
+/**
+ * @brief Evaluate results for 24bit data length
+ *
+ * @param[in] test_case Pointer ot structure defining functionality test case
```

Comment:
Typo here, please correct it in the rest of the doxy docs
```suggestion
 * @param[in] test_case Pointer to structure defining functionality test case
```